### PR TITLE
Update docker-compose

### LINKED ISSUE #59: Update Readme
# 📋 Info

The contents of the Readme are at least partly out of date. Just corrections aside, should probably clarify relation of this version with the former "main".

# 🏁 DoD

- [ ] Updated it

### PR DESCRIPTION
**Pull request by local-minimum**
Makes settings more visible to user.

Includes instructions in readme about the new variables.

@skymandr does this resolve #59 ?

### CODE REVIEW

**Comment by skymandr** [comment]
> @skymandr does this resolve #59 ?

There is a dead link in the readme (the one to GU, didn't check the mailing list, so that may be broken too) and maybe you want to point to this project's wiki instead of old? There is no mention of the relation with this repo and the old one, but if you don't think that is necessary then I guess that is fine. I don't think you can resolve #59 until the link is fixed though.

File: README.md [hunk 1]
```diff
@@ -16,11 +16,14 @@ Gothenburg University is currently buying further development and service from p
...
-## Current focus
+# Setting up and running
 
-The ability to add new Cell Count Calibrations via UI is being developed.
-
-Completing and securing up the code for the first release of 2.x
+1. Clone this library `git@github.com:Scan-o-Matic/scanomatic-standalone.git`
+2. And change directory: `cd scanomatic-standalone`
+3. Build the image `docker-compose build`
+4. Create an environment-file or export the `SOM_PROJECTS_ROOT` and `SOM_SETTINGS` variables. Each pointing to a location where you wish to store your projects and settings respectively. See https://docs.docker.com/compose/environment-variables/#the-env-file for specifying them in an env-file, else include them in your `.bashrc`.
```
Comment (skymandr) [inline README.md:24]:
```suggestion
4. Create an environment-file or export the `SOM_PROJECTS_ROOT` and `SOM_SETTINGS` variables. Each pointing to a location where you wish to store your projects and settings respectively. See https://docs.docker.com/compose/environment-variables/#the-env-file for specifying them in an env-file, else include them in e.g. your `.bashrc`.
```

Comment (skymandr) [reply README.md:24]: maybe (though those who don't use bash probably know that this doesn't apply to them)

**Review by skymandr — CHANGES_REQUESTED** [review]
See my general comment pre-review. The code comment is optional.

**Comment by local-minimum** [comment]
Removed some sections from readme, updated with github status thingy and clarified the environment variables some.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
+[![Scan-o-Matic CI](https://github.com/Scan-o-Matic/scanomatic-standalone/actions/workflows/ci.yml/badge.svg)](https://github.com/Scan-o-Matic/scanomatic-standalone/actions/workflows/ci.yml)
 
 # Scan-o-matic (program) and scanomatic (python module)
 
 This project contains the code for the massive microbial phenotyping platform Scan-o-matic.
 
 Scan-o-matic was published in [G3 September 2016](http://g3journal.org/content/6/9/3003.full).
 
-Please refer to the [Wiki](https://github.com/local-minimum/scanomatic/wiki) for instructions on use, installation and so on.
+Please refer to the general [Scan-o-Matic Wiki](https://github.com/Scan-o-Matic/scanomatic/wiki) for further instructions, though setup and starting should like described below.
```
Comment (skymandr) [inline README.md:9]:
```suggestion
Please refer to the general [Scan-o-Matic Wiki](https://github.com/Scan-o-Matic/scanomatic/wiki) for further instructions, though setup and starting should be as described below.
```

**Review by skymandr — APPROVED** [review]

**Pull request merged**